protocol/v1/qbft/instance/metrics.go

```diff
@@ -16,13 +23,24 @@ var (
 		Name: "ssv:validator:ibft_round",
 		Help: "IBFTs round",
 	}, []string{"identifier", "pubKey"})
+	metricsDurationStageProposal = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "ssv:validator:duration_stage_proposal",
```

Comment:
i think we can combine into 1 with type and pubkey 
no need for metric for each type